src/main/java/net/glowstone/entity/passive/GlowAbstractHorse.java

```diff
@@ -104,6 +123,11 @@ private int getHorseFlags() {
         return BREEDING_FOODS;
     }
 
+    protected int computeTamingAmount(Material material) {
+        // TODO
```

Comment:
Please document what's the intended functionality of the method.

Reply:
Done, and renamed to `computeDomestication` to match the Bukkit nomenclature. Parameter renamed to `food`.